### ISSUE TITLE
Using environment variables to connect via ssh

### ISSUE THREAD

**Issue by shivan-s** [dbcli/pgcli#1433]
## Description
This is not an issue but more of a feature request.

I love this tool. I am wondering if it's possible to look into the environment variables when connecting via ssh.

Say you have a `.env` or a `.env.local` with `DB_URL` as a set environment variable that you can ssh into.

I am happy to contribute if possible but would like some feedback on your thoughts. Thanks in advance.

**Comment by j-bennet** [comment]
@shivan-s Certainly, feel free to contribute! I believe adding `python-dotenv` to the requirements and calling `load_dotenv` would do it.

**Comment by shivan-s** [comment]
Great thank you!

**Comment by dbaty** [comment]
I am a bit late to this (2 years and half)... but I am not sure I understand what you have in mind: 

> `DB_URL` as a set environment variable that you can ssh into.

Given the name, I suppose that `DB_URL` is **not** the name of the remote host you ssh into, is it? Also, is the `.env` file on your computer, or on the remote host (that you ssh into)? A more detailed usage example would help.

**Comment by shivan-s** [comment]
Hi @dbaty, hope you are well.

I've been bad and haven't prioritising a contribution to this project despite my initial enthusiasm. However, I still do love this project and use it daily.

Just for completeness and answering your question. The idea is that if you have an `.env` file in your project (local machine), we have a flag to parse the `.env` file for a variable `DB_URL` and we connect to this.

```
.env
DB_URL=...
```

To be honest, I've been getting by using (using fish as an example):

```fish
pgcli $(grep 'DB_URL' < .env | set s/DB_URI=//)
```

So, I totally understand if you close this issue as totally not worth it.

**Comment by dbaty** [comment]
Ok, got it. I was confused because of the mention of ssh, but it's actually independent. Your comment made that clearer, thanks!

For the record, I do something very similar to what you do with `grep`. You're right that reading from a `.env*` file could be more user-friendly, obviously. We would have to look for `DATABASE_URL`, `DB_URL`, variants with `URI`... and maybe other common names? And hope that the format is consistent. :)

I'll let this issue open. Contributions are most welcome! ;)
